apps/backend/src/modules/groups/group.model.ts

```diff
@@ -0,0 +1,53 @@
+import { Model, type RelationMappings } from "objection";
```

Comment:
Instead of `Model`, I'll use `AbstractModel`, but what about `type RelationMappings`. Shoul I re-export it from other place, or is it okay to import it here from "objection"?